paula-pineros/tarea2/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Nature magazine</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="index.css" />
+        <link rel="stylesheet" href= 
+        "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.css"></link>
+        <style>
+            body {
+               background-image: url("imgs/Background.jpg");
+               background-repeat: no-repeat;
+               background-size: cover;
+            }
+         </style>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <dl>
+                    <dd>Edition 2</dd>
+                    <dd>August 2022</dd>
+                </dl>
+            </nav>
+        </header>
+        <h1> WELLNESS</h1>
+        <section>
+            <menu>
+                <li id="imgs"><img src="imgs/Hiking.jpg" alt="A man hiking"></li> 
```

Comment:
@PaulaPineros , li tags should be under ul tags.

Also use figure tags for all the images